meson_options.txt

```diff
@@ -501,6 +501,8 @@ option('llvm-fuzz', type : 'boolean', value : false,
        description : 'build against LLVM libFuzzer')
 option('kernel-install', type: 'boolean', value: true,
        description : 'install kernel-install and associated files')
+option('ukify', type: 'boolean', value: true,
```

Comment:
Can we please start making new meson options use actual meson feature type instead of bools?

Reply:
I think we'll want to convert all the options to use features (and automatic feature selection) in one fell swoop. I don't think it makes sense to use a mixed approach.

Reply:
I changed the typo to combo 'true'/'false'/'auto' now. The code doesn't work with python3.9 so it's only enabled for >= 3.10.